decksite/auth.py

```diff
@@ -70,9 +72,31 @@ def decorated_function(*args, **kwargs):
 def logout():
     session['admin'] = None
     session['id'] = None
+    session['logged_person_id'] = None
 
 def redirect_uri():
     uri = url_for('authenticate_callback', _external=True)
     if 'http://' in uri:
         os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = 'true'
     return uri
+
+def discord_id():
+    return session.get('id')
+
+def logged_person():
+    return session.get('logged_person_id')
+
+def log_person(person_id):
+    session['logged_person_id'] = person_id
+
+def logged(f):
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        id = discord_id()
```

Comment:
Rename this variable.  `id` is the name of a built-in function, and shouldn't be used as a variable name.

Reply:
This is why Codacy is failing.